lib/pages/login/login_page.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/material.dart';
+import 'package:intl_phone_field/intl_phone_field.dart';
+
+class LoginPage extends StatefulWidget {
+  static const String routeName = '/loginpage';
+
+  const LoginPage({super.key});
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  String dropdownvalue = 'GDSC';
+
+  // List of items in our dropdown menu
+  var items = [
+    'GDSC',
+    'Eco Club',
+    'Item 3',
+    'Item 4',
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    double myWidth = MediaQuery.of(context).size.width;
+    double myHeight = MediaQuery.of(context).size.height;
+    return Scaffold(
+      backgroundColor: Colors.white,
```

Comment:
The styling as discussed shall be defined in the theme
This to ensure a consistent design for the whole app
There is a default light theme which tells the color of Scaffold, If a special scaffold requires a separate we can deal with it later by maybe making a separate widget